runtime/src/lib.rs

```diff
@@ -94,6 +94,12 @@ pub type Moment = u64;
 /// Credential type
 pub type Credential = u64;
 
+/// Represents a thread identifier for both Forum and Proposals Discussion
```

Comment:
Can you here add an extended comment explaining the nature of the problem, and thus why the values MUST indeed be the same currently. That context is lost in the source.

Reply:
done in de4ce1ef37309d96f748358ca429e3667b7176d2